server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/EmailSetMethodContract.scala

```diff
@@ -409,53 +419,43 @@ trait EmailSetMethodContract {
       .asString
 
     assertThatJson(response)
-      .inPath("methodResponses[1][1].list[0]")
-      .isEqualTo(String.format(
-        """{
-          |   "id":"%s",
-          |   "keywords": {
-          |     "$Answered": true
-          |   }
-          |}
-      """.stripMargin, messageId.serialize))
+     .inPath("methodResponses[0][1].notUpdated")
+     .isEqualTo(s"""{
+        | "$invalidMessageId": {
+        |     "type":"serverFail",
```

Comment:
(shouldNotAddKeywordWhenInvalidMessageId)

We should of course have an invalidArgument here